### PR TITLE
Add frontend functionality for business calendars

### PR DESCRIPTION
**Pull request by eash130**

### CODE REVIEW

File: capstone/src/main/webapp/js/business.js [hunk 1]
```diff
@@ -74,6 +76,22 @@ function constructBusinessProfile(id) {
         } else {
           document.getElementById('edit-button').style.display = 'none';
         }
+        
+        const calendarDiv = document.getElementById('business-calendar');
+        if (info.calendarEmail.length > 0) {
```
Comment (lukasberglund) [inline capstone/src/main/webapp/js/business.js:81]:
```suggestion
        if (info.calendarEmail.length != '') {
```
HUGE nit

File: capstone/src/main/webapp/js/business.js [hunk 2]
```diff
@@ -89,3 +107,17 @@ window.toggleProfile = function() {
   viewProfile.style.display = 'none';
   editProfile.style.display = 'block';
 }
+
+window.previewCalendar = function() {
+  const calendar = document.getElementById('edit-calendar-preview');
+  const email = document.getElementById('edit-calendar').value;
+  const warningMessage = document.getElementById('calendar-warning');
+  if (email.length == 0) {
```
Comment (lukasberglund) [inline capstone/src/main/webapp/js/business.js:115]:
```suggestion
  if (email.length != '') {
```

File: capstone/src/main/webapp/business.html [hunk 3]
```diff
@@ -84,8 +85,21 @@ <h3>How You Can Help Us</h3>
...
-              <p>You can share your business website, grubhub delivery link, or describe what your business has to offer!</p>
-              <textarea id="edit-about" name="about" rows="10" cols="30"></textarea>
+              <textarea id="edit-about" name="about" rows="10" cols="30"
+                  placeholder="You can share your business website, grubhub delivery link, or describe what your business has to offer!"></textarea>
+            </div>
+
+            <div id="calendar-section">
+              <p>If you have a google calendar of events, feel free to share the linked email below.</p>
+              <p><strong>NOTE</strong>: Other users may be able to see this email address and events that are not intended to be marked as public.</p>
+              <p>If the email address entered does not have a linked google calendar, it won't display properly</p>
```
Comment (alygerharter) [inline capstone/src/main/webapp/business.html:95]: capitalize "Google Calendar"

File: capstone/src/main/webapp/business.html [hunk 4]
```diff
@@ -84,8 +85,21 @@ <h3>How You Can Help Us</h3>
...
             <div id="about-section">
               <label for="about">Share your business.</label><br>
-              <p>You can share your business website, grubhub delivery link, or describe what your business has to offer!</p>
-              <textarea id="edit-about" name="about" rows="10" cols="30"></textarea>
+              <textarea id="edit-about" name="about" rows="10" cols="30"
+                  placeholder="You can share your business website, grubhub delivery link, or describe what your business has to offer!"></textarea>
+            </div>
+
+            <div id="calendar-section">
+              <p>If you have a google calendar of events, feel free to share the linked email below.</p>
```
Comment (winniewan) [inline capstone/src/main/webapp/business.html:93]: Will non business users be able to have a calendar to show?

Comment (eash130) [reply capstone/src/main/webapp/business.html:93]: No

File: capstone/src/main/webapp/business.html [hunk 5]
```diff
@@ -84,8 +85,21 @@ <h3>How You Can Help Us</h3>
...
+              <p><strong>NOTE</strong>: Other users may be able to see this email address and events that are not intended to be marked as public.</p>
+              <p>If the email address entered does not have a linked google calendar, it won't display properly</p>
+              <input id="edit-calendar" type="text" name="calendarEmail" onchange="previewCalendar()"></input>
```
Comment (winniewan) [inline capstone/src/main/webapp/business.html:96]: Can you enter an email address that is not used to sign in to the website?

Comment (eash130) [reply capstone/src/main/webapp/business.html:96]: Yes

**Pull request merged**